tests/functions/test_origin_request.py

```diff
@@ -74,6 +79,7 @@
         "no alias keywords",
         "releasever alias",
         "layered rhui, releasever alias",
+        "no content_type"
```

Comment:
Similarly as the comment against reftest, this does not cover the realistic case properly.  Realistically, items can have no content_type at all which is different from having a content_type of an empty string.  In fact the old code would already not crash if an item had content_type of `""`, so adding a test for that value is not going to prove that the crash was fixed.

I think you ought to edit the test (or add a new one) which truly tests the behavior on *no* content_type, rather than a content_type of `""`.